sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1345,6 +1345,16 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val LITERAL_PRECISE_PRECISION =
```

Comment:
mmh... PRECISE_PRECISION sounds weird... can we look for a better one? I don't have a suggestion right now, but I'll think about it.